roles/pmk-selinux/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+- copy:
+    src: "{{ role_path }}/files/pf9.te"
```

Comment:
You should not need `role_path`. Ansible will automatically look for pf9.te under files/ in the role's directory.

Reply:
interesting, i tried that - it wasn't copying them over automatically (vagrantfile has an example to repro).